scripts/env.sh

```diff
@@ -94,8 +124,19 @@ NUM_VFS=${NUM_VFS:-"46"}
 
 # Feature Configuration
 
+# ArgoCD Configuration
+ARGOCD_CHART_VERSION=${ARGOCD_CHART_VERSION:-"7.8.2"}
+
+# Maintenance Operator Configuration
+MAINTENANCE_OPERATOR_VERSION=${MAINTENANCE_OPERATOR_VERSION:-"0.2.0"}
+
 # Hypershift Configuration
-HYPERSHIFT_IMAGE=${HYPERSHIFT_IMAGE:-"quay.io/hypershift/hypershift-operator:latest"}
+ENABLE_HCP_MULTUS=${ENABLE_HCP_MULTUS:-"true"}
+if [ "${ENABLE_HCP_MULTUS}" = "true" ]; then
```

Comment:
Why it is required? Use the one with latest changes, i believe we can use master